app/javascript/src/stores/editor.js

```diff
@@ -10,7 +10,7 @@ import { debounced } from "../utils/debounceStore"
 const VARIABLE_EXTRACTION_DEBOUNCE_MS = 500
 
 export const screenWidth = writable(0)
-export const isMobile = derived(screenWidth, $screenWidth => $screenWidth && $screenWidth < 1100)
+export const isMobile = derived(screenWidth, $screenWidth => $screenWidth && $screenWidth < 999)
```

Comment:
There is this state in-between 1000 and 1100 pixels wide where the Editor sections are sorted horizontally, but you have to drag them vertically. Any clue why that might be?

Reply:
i know that the drag-direction is set in DragHandle.svelte with `$: direction = $isMobile ? "pageY" : "pageX"`, and isMobile begins at 1100px. the question is where the editor sections become horizontal at 1000px. They really should match the same width.

Reply:
> and isMobile begins at 1100px

Doesn't this change make it start at 1000px, not 1100px?

Reply:
We can change $isMobile to start at 1000 to match the styling

Reply:
yeah thats what i intended, but are you sure it wont change any other interactions expecting 1100? or maybe thats the only 1100

Reply:
This store is only used in the editor. Everything else relies on the media query, which is already 1000. So it was probably an unintentional mismatch. At worst the 1000-1100 range would be less optimal in terms of UX, but we will see.